chainservice/builder_test.go

```diff
@@ -20,13 +20,25 @@ func TestEstimateTipHeight(t *testing.T) {
 		blk, err := block.NewBuilder(block.RunnableActions{}).SetHeight(cfg.Genesis.DardanellesBlockHeight + 1).SignAndBuild(sk)
 		r.NoError(err)
 		r.Equal(blk.Height()+2, estimateTipHeight(&cfg, &blk, 10*time.Second))
+		blk, err = block.NewBuilder(block.RunnableActions{}).SetHeight(cfg.Genesis.WakeBlockHeight - 3).SignAndBuild(sk)
+		r.NoError(err)
+		r.Equal(blk.Height()+2, estimateTipHeight(&cfg, &blk, 10*time.Second))
+		r.Equal(blk.Height()+3, estimateTipHeight(&cfg, &blk, 13*time.Second))
+		r.Equal(blk.Height()+3, estimateTipHeight(&cfg, &blk, 15*time.Second))
 	})
 	t.Run("before dardanelles", func(t *testing.T) {
 		blk, err := block.NewBuilder(block.RunnableActions{}).SetHeight(cfg.Genesis.DardanellesBlockHeight - 100).SignAndBuild(sk)
 		r.NoError(err)
 		r.Equal(blk.Height()+1, estimateTipHeight(&cfg, &blk, 10*time.Second))
 		blk, err = block.NewBuilder(block.RunnableActions{}).SetHeight(cfg.Genesis.DardanellesBlockHeight - 1).SignAndBuild(sk)
 		r.NoError(err)
-		r.Equal(blk.Height()+3, estimateTipHeight(&cfg, &blk, 20*time.Second))
+		r.Equal(blk.Height()+4, estimateTipHeight(&cfg, &blk, 20*time.Second))
```

Comment:
Fix previous bugs: the block interval at dardanelles hard fork height used 10s instead of 5s.